### ISSUE TITLE
W10 Tutorial DG Review

### ISSUE THREAD

**Issue by J-Dan23** [AY1920S2-CS2103T-W12-1/main#210]
Setting up:
![image](https://user-images.githubusercontent.com/55853397/77718603-4d834600-701e-11ea-8054-1a28cafb40b2.png)
Under 3.: Still named as "seedu.address.Main" Should it be renamed to match your morphed application? Check if there are other yet to be renamed path names.

General comments:
For consistency reasons, check if any of your diagrams are missing figure labels/descriptors.
Nice, consistent use of case for the headers. Good job!
Bold markdown words might be more suitable as code markdowns instead, such as **postal sector** could be `postal sector` instead.
Some figures are lacking names, while other have them.
Subsectioning and hyperlinks make the DG very easy to navigate.

**Comment by J-Dan23** [comment]
Design:

2.1
![image](https://user-images.githubusercontent.com/55853397/77718503-101eb880-701e-11ea-9ff5-b9c6e367aefd.png)
Bug in Fig 3: Should the return path from :Storage to Model for [when deleting return order] be outside the alternate path box?

2.1
![image](https://user-images.githubusercontent.com/55853397/77718531-1f9e0180-701e-11ea-8a40-2f3b53049415.png)
Bug: Should PersonListPanel be mentioned in the description? There might be similar bugs in subsequent sections where components and methods are not renamed.

**Comment by J-Dan23** [comment]
2.4. Model

![image](https://user-images.githubusercontent.com/55853397/77718738-94713b80-701e-11ea-9785-55bef0c32878.png)
For consistency reasons, headers should have the same format. In this case do you plan to call them all "The xxx component," or simply "The xxx,"?
Additionally, if "Order Book" is an object, should it be marked down? Check if there are other similar mark down issues in your DG.

![image](https://user-images.githubusercontent.com/55853397/77718813-bff42600-701e-11ea-8085-0873bbeadca5.png)
Bug: BetterModelClassDiagram not displaying.

**Comment by J-Dan23** [comment]
2.2. Storage Component

![image](https://user-images.githubusercontent.com/55853397/77719050-4f99d480-701f-11ea-8001-3299314fb334.png)
For consistency reasons, should Fig 8 also be surrounded by a box labelled "Storage" as well? Check if there are other diagrams similarly lacking in component boxes.

**Comment by J-Dan23** [comment]
3.1. Delivered feature

![image](https://user-images.githubusercontent.com/55853397/77719161-92f44300-701f-11ea-9521-e990c1f4a085.png)
Our team greatly appreciates the class diagram. However, it might be even better to make use of your class diagrams to explain how a feature works (in this case, the delete feature).
You may want to make the notation adhere to UML notation, though, to make it even more readable.

![image](https://user-images.githubusercontent.com/55853397/77719910-6ccfa280-7021-11ea-86d0-3bd2763dd0d4.png)
Odd line length formatting here.

**Comment by J-Dan23** [comment]
3.2 Nearby feature

![image](https://user-images.githubusercontent.com/55853397/77719486-57a64400-7020-11ea-8e8b-08bcd5c473bc.png)
The class diagram can be a little overwhelming. Thus, maybe consider simplifying both the diagram and the code by using a hashmap of variables with the key being the district number perhaps? Alternatively, you could collate the variables using "districtN" or "addDistrictN".
In summary, perhaps an architecture diagram might be more useful here, as the information would be more concise and clearer due to reduced stating of methods and variables.
